src/pyroboplan/models/panda.py

```diff
@@ -26,6 +28,33 @@ def load_models(use_sphere_collisions=False):
     return pinocchio.buildModelsFromUrdf(urdf_filepath, package_dirs=models_folder)
 
 
+def load_point_cloud(pointcloud_path=None):
+    """
+    Loads a point cloud from a PLY file and converts it into an octree structure.
+
+    Returns
+    -------
+    octree : fcl.Octree
+        An octree data structure representing the hierarchical spatial partitioning
+        of the point cloud. The leaf size (voxel resolution) is set to 0.01 units.
+    """
+    if pointcloud_path is None:
+        models_folder = get_example_models_folder()
+        pointcloud_path = os.path.join(
+            models_folder, "example_point_cloud", "example_point_cloud.ply"
+        )
+
+    # Read the PLY file
+    ply_data = PlyData.read(pointcloud_path)
+
+    # Access vertex data
+    vertices = ply_data["vertex"]
+    vertex_array = np.array([vertices["x"], vertices["y"], vertices["z"]]).T
+    octree = fcl.makeOctree(vertex_array, 0.01)
```

Comment:
I'd personally make this into an optional argument so users can change it, and also default to a bigger value -- for example, I used 0.05 and it fills all the "holes":

![image](https://github.com/user-attachments/assets/7c72439e-7018-40e3-9e17-ac6f896f2e4b)